core/trino-main/src/main/java/io/trino/operator/aggregation/state/Int128StateSerializer.java

```diff
@@ -13,58 +13,57 @@
  */
 package io.trino.operator.aggregation.state;
 
-import io.airlift.slice.Slice;
-import io.airlift.slice.Slices;
 import io.trino.spi.block.Block;
 import io.trino.spi.block.BlockBuilder;
 import io.trino.spi.function.AccumulatorStateSerializer;
+import io.trino.spi.type.DecimalType;
+import io.trino.spi.type.Decimals;
 import io.trino.spi.type.Int128;
 import io.trino.spi.type.Type;
 
-import static io.trino.spi.type.VarbinaryType.VARBINARY;
 
-public class LongDecimalWithOverflowStateSerializer
-        implements AccumulatorStateSerializer<LongDecimalWithOverflowState>
+public class Int128StateSerializer
+        implements AccumulatorStateSerializer<Int128State>
 {
-    private static final int SERIALIZED_SIZE = Long.BYTES + Int128.SIZE;
+    private static final int SERIALIZED_SIZE = Long.BYTES + Int128.SIZE * 2;
```

Comment:
should this be just `Int128.SIZE`?

Reply:
Yes, obviously yes. It is a quirky mistake